fastqfil/bio_files_processor.py

```diff
@@ -0,0 +1,11 @@
+#Creates a file with combined sequences
+def convert_multiline_fasta_to_oneline(input_fasta, output_fasta = "oneline_fasta.fasta"):

```

Comment:
Голову функции можно улучшить, если добавить аннотацию типов
```suggestion
def convert_multiline_fasta_to_oneline(input_fasta: str, output_fasta: str = "oneline_fasta.fasta"):
```